docs/source/legacy-lrgs-userguide.rst

```diff
@@ -158,8 +158,9 @@ to a variety of programs shown at the right.
 
 The core of the LRGS software was written by ILEX Engineering, Inc.,
 under contract to the federal government. As such it is freely
-available. Cove Software, LLC now maintains the software and provides
-support to the user community.
+available. The software is now hosted at https://github.com/opendcs/opendcs 
+and maintained by the current OpenDCS maintainers and anyone else who 
+wishes to contribute.
```

Comment:
Oh, `If paid support is required or desired see https://github.com/opendcs/opendcs/wiki/Getting-Paid-Support for known providers`